packages/calcite-components/src/components/alert/alert.stories.ts

```diff
@@ -8,171 +8,224 @@ export default {
   parameters: {
     chromatic: {
       delay: 500,
+      disableSnapshot: false,
     },
   },
 };
 
+const wrapper = html`
```

Comment:
These are wrapper styles or common story styles, so could you rename this to match? It might also be good to note why we need this by adding a comment pointing to https://www.chromatic.com/docs/snapshots/#why-isn%E2%80%99t-my-modal-or-dialog-captured.